Problem with mcAddress on payara micro

I want to change multicast address group of Hazelcast.
But some errors are raised. Is this correct work?

payara-version: payara-micro-4.1.1.154.jar 
java-version: Java HotSpot(TM) 64-Bit Server VM (build 25.60-b23, mixed mode)
OS: Mac OS 10.11.1

```
% java -jar ~/Downloads/payara-micro-4.1.1.154.jar --deploy target/example-payara_micro.war --mcAddress 224.2.2.4 --mcPort 2904
Dec 07, 2015 8:23:52 PM org.glassfish.security.services.impl.authorization.AuthorizationServiceImpl initialize
INFO: Authorization Service has successfully initialized.
Dec 07, 2015 8:23:52 PM org.hibernate.validator.internal.util.Version <clinit>
INFO: HV000001: Hibernate Validator 5.1.2.Final
Exception in thread "main" fish.payara.micro.BootstrapException: PlainTextActionReporterFAILURENo configuration found for server.hazelcast-runtime-configuration
	at fish.payara.micro.PayaraMicro.bootStrap(PayaraMicro.java:714)
	at fish.payara.micro.PayaraMicro.main(PayaraMicro.java:105)
Caused by: org.glassfish.embeddable.GlassFishException: PlainTextActionReporterFAILURENo configuration found for server.hazelcast-runtime-configuration
	at com.sun.enterprise.glassfish.bootstrap.ConfiguratorImpl.configure(ConfiguratorImpl.java:75)
	at com.sun.enterprise.glassfish.bootstrap.GlassFishImpl.configure(GlassFishImpl.java:71)
	at com.sun.enterprise.glassfish.bootstrap.GlassFishImpl.<init>(GlassFishImpl.java:65)
	at com.sun.enterprise.glassfish.bootstrap.StaticGlassFishRuntime$1.<init>(StaticGlassFishRuntime.java:116)
	at com.sun.enterprise.glassfish.bootstrap.StaticGlassFishRuntime.newGlassFish(StaticGlassFishRuntime.java:116)
	at fish.payara.micro.PayaraMicro.bootStrap(PayaraMicro.java:694)
```

--- a/nucleus/payara-modules/payara-micro/src/main/java/fish/payara/micro/PayaraMicro.java
+++ b/nucleus/payara-modules/payara-micro/src/main/java/fish/payara/micro/PayaraMicro.java
@@ -523,26 +523,24 @@ public class PayaraMicro {
             throw new IllegalStateException("Payara Micro is already running, calling bootstrap now is meaningless");
         }       
         // check hazelcast cluster overrides
-        if (!noCluster) { // ie we are clustering
-            MulticastConfiguration mc = new MulticastConfiguration();
-            mc.setMemberName(instanceName);
-            if (hzPort > Integer.MIN_VALUE) {
-                mc.setMulticastPort(hzPort);
-            }
-
-            if (hzStartPort > Integer.MIN_VALUE) {
-                mc.setStartPort(hzStartPort);
-            }
-
-            if (hzMulticastGroup != null) {
-                mc.setMulticastGroup(hzMulticastGroup);
-            }
-            
-            if (alternateHZConfigFile != null) {
-                mc.setAlternateConfiguration(alternateHZConfigFile);
-            }
-            HazelcastCore.setMulticastOverride(mc);
+        MulticastConfiguration mc = new MulticastConfiguration();
+        mc.setMemberName(instanceName);
+        if (hzPort > Integer.MIN_VALUE) {
+            mc.setMulticastPort(hzPort);
         }
+
+        if (hzStartPort > Integer.MIN_VALUE) {
+            mc.setStartPort(hzStartPort);
+        }
+
+        if (hzMulticastGroup != null) {
+            mc.setMulticastGroup(hzMulticastGroup);
+        }
+
+        if (alternateHZConfigFile != null) {
+            mc.setAlternateConfiguration(alternateHZConfigFile);
+        }
+        HazelcastCore.setMulticastOverride(mc);
         
         setSystemProperties();
         BootstrapProperties bprops = new BootstrapProperties();
@@ -673,14 +671,6 @@ public class PayaraMicro {
                     gfproperties.setConfigFileReadOnly(false);
                 }
 
-            }
-
-            if (this.hzPort != Integer.MIN_VALUE) {
-                gfproperties.setProperty("embedded-glassfish-config.server.hazelcast-runtime-configuration.multicastPort", Integer.toString(hzPort));
-            }
-
-            if (this.hzMulticastGroup != null) {
-                gfproperties.setProperty("embedded-glassfish-config.server.hazelcast-runtime-configuration.multicastGroup", hzMulticastGroup);
             }
 
             if (this.maxHttpThreads != Integer.MIN_VALUE) {

